notarization_poller/src/notarization_poller/config.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+"""Config for scriptworker.
+
+Attributes:
+    DEFAULT_CONFIG (frozendict): the default configuration
+    log (logging.Logger): the log object for the module.
+
+"""
+import argparse
+import logging
+import logging.handlers
+import os
+from copy import deepcopy
+
+from frozendict import frozendict
+from yaml import safe_load
+
+from notarization_poller.constants import DEFAULT_CONFIG
+from notarization_poller.exceptions import ConfigError
+
+log = logging.getLogger(__name__)
+
+
+def update_logging_config(config, log_name=None, file_name="worker.log"):
```

Comment:
Hm. I think we avoided formatting the logging from the script too much, to avoid double timestamps and the like (`worker_timestamp worker_level - worker_msg` : the `worker_msg` is the task output, so with full script log formatting we would get `worker_timestamp worker_level - script_timestamp script_level - script_msg`.  Also, we don't have to worry about rotating file logs in a script, because it's not long-running.

Reply:
I ended up calling `_init_logging` in this function :)